cl/recap/views.py

```diff
@@ -101,6 +122,15 @@ class PacerFetchRequestViewSet(LoggingMixin, ModelViewSet):
         "date_modified",
         "date_completed",
     )
+    # Default cursor ordering key
+    ordering = "-id"
+    # Additional cursor ordering fields
+    cursor_ordering_fields = [
+        "id",
+        "date_created",
+        "date_modified",
+        "date_completed",
```

Comment:
I noticed that "date_completed" seems to be a supported key for cursor pagination. However, the `compatible_sorting` dictionary within the `VersionBasedPagination` class doesn't include this key. This might lead to a `KeyError` exception during key lookup in the `handle_database_cursor_pagination` method (`self.compatible_sorting[requested_ordering]`), or am i missing something?

Reply:
Thanks for noticing this issue. I've addressed it and improved the tests to catch these kinds of errors.